guides/m1x/ce19-ee114/ce1.9_release-notes.html

```diff
@@ -60,6 +61,42 @@ <h2 id="upgrade">Important Upgrade Information</h2>
+ <p>This version (or patch SUPEE-10975, which applies to older versions of Magento) provides resolution of multiple critical security issues and functional fixes. These critical security issues include remote code execution (RCE),  cross-site scripting (XSS), and cross-site request forgery (CSRF) issues. This release also provides support for PHP 7.2.</p>
+
+<p>We recommend upgrading your Magento store to this latest version.  See  <a href="https://magento.com/security/patches/supee-10888" target="_blank">Magento Security Center</a> for a comprehensive discussion of these issues.</p>
+
+<p>Note that after updating to this release, third-party modules that depend upon Magento core backup functionality will no longer work. Alternatively, you can use one of these two methods  to enable database backups: </p>
+
+<ul>
+<li>Change <b>false</b> to <b>true</b> in the <b>config</b> > <b>modules</b> > <b>Mage_Backup</b> > <b>active</b>  setting of <code> app/etc/modules/Mage_All.xml</code>. </li>
+<li>From the Admin,  change the <b>System</b> > <b>Configuration</b> > <b>Advanced</b> > <b>Disable  Modules Output</b> > <b>Mage_Backup</b> from <b>disabled</b> to <b>enabled</b>.</li>
+</ul>
+
+<h3>Fixed issues and enhancements</h3>
+<ul>
+<li><!-- MPERF-10223 -->This release provides support for PHP 7.2. </li>
+<li><!-- MPERF-10217 -->We’ve removed the CC module. As a result, third-party modules that depend upon either the <code>ccsave</code> method or the <code>xmlconnect</code> method will not work as expected.  Third-party themes that implement <code>ccsave</code> will not work as expected, either. </li>
+<li><!-- MPERF-10219 10218 -->The Magento logo has been updated throughout the code base. </li>	
+<li><!-- MPERF-8829 -->The <b>Continue</b> button now works as expected on the Payments step of checkout  when paying with the PayPal payment method. </li>	
+<li><!-- MPERF-8826 -->Google Tag Manager now logs sales information in Google Analytics as expected. </li>	
+<li><!-- MPERF-8382 -->The product export CSV file now contains columns for super attributes. </li>	
+<li><!-- MPERF-7181 -->Magento no longer throws an error when a customer accesses their shopping cart after items in their cart have been removed due to a timeout. Previously, Magento displayed this error, `Notice: Undefined variable: freePackageValue in /var/www/dev/htdocs/app/code/core/Mage/Shipping/Model/Carrier/Tablerate.php on line 130`.  </li>	
+<li><!-- MPERF-7067 -->Clicking on a configurable product’s swatch on the  product list page now updates product price as expected.  </li>	
+<li><!-- MPERF-6629 -->Customers can now successfully add a grouped product to their shopping cart when category permissions are enabled. Previously, Magento did not add the product to the cart, but instead displayed a descriptive error message.  </li>	
+<li><!-- MPERF-6566 -->Magento no longer displays incorrect prices on the storefront after a failure of the enterprise refresh index.  </li>	
```

Comment:
Verify whether "enterprise refresh index" belongs in the CE release notes.

Reply:
Verified with PM that this note belongs in both CE and EE.